ui/next/ts/app.tsx

```diff
@@ -0,0 +1,14 @@
+/// <reference path="../../typings/main.d.ts" />
+
+import * as React from "react";
+import * as ReactDOM from "react-dom";
```

Comment:
Typescript 1.8 prefers ES6 syntax now for imports; the 'import *' is necessary because there is no default export from react.